cfme/markers/stream_excluder.py

```diff
@@ -67,17 +67,20 @@ def pytest_itemcollected(item):
                 item.add_marker(pytest.mark.uncollect)
 
 
-def pytest_collection_modifyitems(session, config, items):
+def pytest_sessionstart(session):
+    config = session.config
     # Just to print out the appliance's streams
     from fixtures.terminalreporter import reporter
```

Comment:
**QUESTION:** Can't this be imported up at the top?

Reply:
i didn't experiment, i'd like to leave it for a follow-up